tools/bin/prep_test_results_for_gcs.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import json
+import os
+
+
+'''
+
+This script is intended to be run in conjuction with https://github.com/EnricoMi/publish-unit-test-result-action to upload trimmed
+test results from the output to a GCS bucket for further analysis.
+
+The script takes as input the filename of the json output by the aforementioned action, trims it, and uploads it to GCS with a ".jsonl" filename 
+
+'''
+
+# Initiate the parser
+parser = argparse.ArgumentParser()
+
+# Add long and short argument
+parser.add_argument("--json", "-j", help="Path to the result json output by https://github.com/EnricoMi/publish-unit-test-result-action")
+
+def main():
+    # Read arguments from the command line
+    args = parser.parse_args()
+
+    token = os.getenv('GITHUB_TOKEN')
+
+    f = open(args.json)
+    d = json.load(f)
+    out = []
+    
+    check_run_id = int(d["check_url"].split("/")[-1])
```

Comment:
Same comment as https://github.com/airbytehq/airbyte-cloud/pull/3205/files#r1010647880
It would be great if we could avoid this duplication.

Reply:
👍